docs/template-development.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Template development"
+weight: 50
+---
+
+## Minimum template requirements
+
+Let's discuss the minimum components your template requires: `template` directory and `package.json` file.
+
+## 1. `template` directory
```

Comment:
```suggestion
### `template` directory
```

Reply:
Are you sure you accepted all changes @pratik2315 ? I am still seeing numbers 1 and 2 in these headers when I display the rich diff view